js/webview/swipeEvents.js

```diff
@@ -9,15 +9,27 @@ window.addEventListener("mousewheel", function (e) {
 
 	verticalMouseMove += e.deltaY;
 	eventsCaptured++;
+	platformZoomKey = e.ctrlKey;
 
+	if (navigator.platform == "MacIntel") {
+		if(e.ctrlKey) {
```

Comment:
This should also check that ```e.defaultPrevented``` is ```false```. Otherwise, things like the pdfjs viewer break, since they have their own pinch-to-zoom code.